auth_app/models.py

```diff
@@ -35,3 +39,24 @@ class User(AbstractBaseUser, PermissionsMixin):
 
     def __str__(self):
         return "@{}".format(self.username)
+
+
+@receiver(post_save, sender=User)
```

Comment:
Isn't there another part of the flow where you need to create a bot for a specific user?

Reply:
In the future, it will be an option to add bots.